### PR TITLE
Style password reset and other registration forms. Fixes #478

### LINKED ISSUE #478: Need styling on the password reset and forgotten username pages

### PR DESCRIPTION
**Pull request by jfrankl**

### CODE REVIEW

File: src/nyc_trees/apps/login/templates/login/forgot_username.html [hunk 1]
```diff
@@ -1,16 +1,15 @@
-{% extends "base.html" %}
+{% extends "../templates/registration/registration_base.html" %}
```
Comment (maurizi) [inline src/nyc_trees/apps/login/templates/login/forgot_username.html]: You should be able to remove the `../templates/` here I think.

**Comment by kdeloach** [comment]
Looks the same to me.

image

**Comment by kdeloach** [comment]
Disregard that, I was on the wrong branch!

**Comment by kdeloach** [comment]
I don't see styles on this branch either. I can show you when you're available.

**Comment by jfrankl** [comment]
This branch isn't going to make the Register page look different. Instead, it applies the standard registration/login template to a bunch of little pages.

#486 will style the Register page.

**Comment by kdeloach** [comment]
Cool, looks like everything still works.

**Comment by kdeloach** [comment]
+1

**Pull request merged**